get-latest-pulsecore.sh

```diff
@@ -21,28 +21,35 @@ if [ ! -d "$BUILDERDIR" ]; then
     exit 1
 fi
 
-if [ -z "$LATEST_FEDORA_RELEASE" ]; then
-    LATEST_FEDORA_RELEASE="$(git ls-remote --heads https://src.fedoraproject.org/rpms/fedora-release | grep -Po "refs/heads/f[0-9][1-9]*" | sed 's#refs/heads/f##g' | sort -g | tail -1)"
-fi
-LATEST_FEDORA_VERREL="$(dnf -q repoquery pulseaudio --disablerepo=* --enablerepo=fedora --enablerepo=updates --releasever="$LATEST_FEDORA_RELEASE" | grep -Po "[0-9][1-9]*\.*[0-9]*\.*[0-9]*-[0-9]*" | sort -V | tail -1)"
+REPO_URL=git://anongit.freedesktop.org/pulseaudio/pulseaudio
```

Comment:
Please https url.

Reply:
Looks like gitlab is the source of truth for other ones (anongit, cgit) so changed url to that one instead.